configs/opa_bundles/default/authz.rego

```diff
@@ -46,13 +46,28 @@ allow if {
 	input.user_id == input.resource_id
 }
 
+allow if {
+	authn
+	input.kind == "usr"
+	input.action == "get-org-member-status"
+	member_of_single_assosicated_org_id
+}
+
+# anyone can translate an email to an id.
+allow if {
+	authn
+	input.kind == "usr"
+	input.action = "get-id"
+}
+
 #
 # Orgs
 #
 
 allow if {
 	authn
 	input.kind == "org"
+	input.action in ["delete", "update", "remove-member"]
```

Comment:
So any member of an org can take over the org by removing all other members, or even delete the org?

It's bad but not a merge blocker because we don't support RBAC yet, and in fact being specific about actions is better than before. I just want to confirm I'm understanding this correctly.